### DISCUSSION TITLE
求中世纪2全面战争

### DISCUSSION THREAD

**Discussion by ds4275364** [Gamuxorg/bbs#197] [投递和请求游戏]
求中世纪2全面战争

**Comment by shouhuanxiaoji** [comment]
magnet:?xt=urn:btih:DC60A6DE2B3AF6ABE87C5D8AA705971C08FCABB4&tr=http%3A%2F%2Fbt2.t-ru.org%2Fann%3Fmagnet&dn=Medieval%20II%3A%20Total%20War%E2%84%A2%20%5Bx86%5D%20%5BRUS%20%2F%20ENG%5D%20%5BNative%5D
太大了。给个链接，自己用bt软件来吧

**Comment by ds4275364** [comment]
下不下来....
